common/validators.js

```diff
@@ -97,5 +97,42 @@
                 // });
             }
         };
+    })
+
+    .directive('timestamp', function() {
+        return {
+            restrict: 'A',
```

Comment:
I actually meant it the other way around where this would be a child of `<input>`.  As in:
```
<input>
    <timestamp validate-values>
</input>
```
I'm more curious if you thought about this as a possible solution rather than adding an attribute directive that also relies on another attribute.